Lib/sysconfig.py

```diff
@@ -658,6 +658,15 @@ def get_config_vars(*args):
         _CONFIG_VARS['platbase'] = _EXEC_PREFIX
         _CONFIG_VARS['projectbase'] = _PROJECT_BASE
         _CONFIG_VARS['platlibdir'] = sys.platlibdir
+
+        # when we are not in virtual environment or RPM build
+        # we change '/usr/'  to '/usr/local'
+        # to avoid surprises, we explicitly check for the hardcoded values
```

Comment:
Can we be explicit about what hardcoded values are in question?

Reply:
We are later on when this is redone in the [Fixup: Only change the defaults for prefixes for the posix_prefix scheme](https://github.com/fedora-python/cpython/pull/48/commits/9d74bdf33e7ec25c13346ea1b5f6ac6c6ebdfba4) commit.